code/modules/mob/living/carbon/human/say.dm

```diff
@@ -219,3 +223,78 @@
 		returns[2] = 50
 		return returns
 	return ..()
+
+/mob/living/carbon/human/hear_say(var/message, var/verb = "says", var/datum/language/language = null, var/alt_name = "",var/italics = 0, var/mob/speaker = null)
+	for(var/T in get_traumas())
+		var/datum/brain_trauma/trauma = T
+		if(!trauma.suppressed)
+			message = trauma.on_hear(message, verb, language, alt_name, italics, speaker)
+	..()
+
+/mob/living/carbon/human/proc/handle_speech_muts(var/message, var/verb)
+	if(message)
+		if(disabilities & TOURETTES)

```

Comment:
This is not at all what tourettes is.

Reply:
This just copies the already extant Tourettes mechanics and expands them to the word filter system.

Reply:
Adding onto this, I fail to see how tourettes is "not at all" this.